src/ca65/instr.c

```diff
@@ -434,7 +440,7 @@ static const struct {
         { "INY",  0x0000001, 0xc8, 0, PutAll },
         { "JMP",  0x0010808, 0x4c, 6, PutAll },
         { "JSR",  0x0000008, 0x20, 7, PutAll },
-        { "LDA",  0x080A66C, 0xa0, 0, PutAll },
+        { "LDA",  0x080A26C, 0xa0, 0, PutAll },
```

Comment:
`LDA (zp)` is a valid addressing mode for that instruction on CMOS chips.  Some libraries use it.  That change stops those libraries from being built.  That's why the Travis CI test failed.

Reply:
This looks like a typo because it is in the 65c02 struct which i didn't want to change.